usaspending_api/accounts/views/financial_spending.py

```diff
@@ -24,7 +23,7 @@ def get_queryset(self):
 
         # required query parameters were not provided
         if not (fiscal_year and agency_id):
-            raise InvalidParameterException('Missing one or more required query parameters: fiscal_year, agency_id')
+            raise ParseError('Missing one or more required query parameters: fiscal_year, agency_id')
```

Comment:
Oooh, good sleuthing. Should we fix that? Asking because I'm using `InvalidParameterException` for the `financial_balances/agencies` endpoint and want to be consistent. But it doesn't seem right to return a 500 for bad params.